ui/v2.5/src/components/Performers/PerformerDetails/PerformerDetailsPanel.tsx

```diff
@@ -213,7 +213,7 @@ export const PerformerDetailsPanel: React.FC<IPerformerDetails> = ({
             fullWidth={fullWidth}
           />
           <DetailItem
-            id="StashIDs"
+            id="stash_ids"
```

Comment:
I was actually under the impression that StashID wasn't meant to be translated. I didn't realize a location string existed for it.

Reply:
I only noticed because I was getting a console error about a missing intl string - `react-intl`'s behaviour of just showing the ID when no intl string exists isn't meant to actually be used, it's just a fallback so that at least something is shown.